aws-sandbox-01.tfvars

```diff
@@ -0,0 +1,7 @@
+aws_account_id=922775083351
```

Comment:
went with different variables files for each environment, named after the AWS accounts they'll be deployed into. Should this be handled using workspaces instead?